src/store/ChartStore.ts

```diff
@@ -741,6 +741,7 @@ class ChartStore {
         } = props;
         this.feedCall = feedCall || {};
         this.api = new BinaryAPI(requestAPI, requestSubscribe, requestForget, requestForgetStream);
+        this.currentLanguage = localStorage.getItem('current_chart_lang')?.toLowerCase();
```

Comment:
When we load for the first time, `this.currentLanguage` will be undefined, right?

Reply:
Yes, correct, that's why there will be one API call to set the language for the first time. We can't really do anything, as without this, there's a language bug which will not change language for market switcher